_stbt/utils.py

```diff
@@ -88,8 +89,51 @@ def find_import_name(filename):
     return import_dir, import_name
 
 
+if sys.version_info.major == 2:  # Python 2
+    text_type = unicode  # pylint: disable=undefined-variable
+
+    def strip_newtypes(text):
```

Comment:
This could all probably be replaced by [future.utils.native](https://python-future.org/reference.html#future.utils.native):

> On Py3, this is a no-op: native(obj) -> obj
>
> On Py2, returns the corresponding native Py2 types that are superclasses for backported objects from Py3:
> [...]
> Existing native types on Py2 will be returned unchanged.